01_dj_webinar.ipynb

```diff
@@ -0,0 +1,1288 @@
+{
```

Comment:
<p>Will you enter the code or will they?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sinzlab/datajoint-webinar/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
They will probably enter the code alongside me most of the time but in some cases they could also do it themselves.